locale/es.po

```diff
@@ -1242,3 +1235,19 @@ msgstr "En espera durante {} segundos..."
 #: pikaur/urllib_helper.py:106
 msgid "pikaur requires python-pysocks to use a socks5 proxy."
 msgstr "pikaur requiere python-pysocks para usar un proxy socks5."
+
+#, python-brace-format
+#~ msgid "Can't copy '{from_path}' to '{to_path}'."
```

Comment:
i see i18n got removed these lines from the translation, i've fixed that, please git pull to the latest fix: e17e045be14530b8696d9cf64f8041e117172928